src/subscriber.go

```diff
@@ -104,3 +108,22 @@ func (s *Subscriber) Unsubscribe() {
 
 	s.logger.Info("unsubscribed")
 }
+
+func (s *Subscriber) forwardRTCP() {
+	if s.Track.Kind() != webrtc.RTPCodecTypeVideo {
+		return
+	}
```

Comment:
Nit: should we not check it before calling `forwardRTCP()`? I.e. if we know that there are not RTCPs to forward for a given track, perhaps we don't necessary need to start `forwardRTCP()` in a new go-routine?